File: Python/OOPs/Inheritance/multiple_inheritance.py
class Camera:
    def take_photo(self):
        print("Photo captured 📷")

class MusicPlayer:
    def play_music(self):
        print("Playing music 🎵")

class Smartphone(Camera, MusicPlayer):
    def browse_internet(self):
        print("Browsing the internet 🌐")

# Test
phone = Smartphone()
phone.take_photo()        # Inherited from Camera
phone.play_music()        # Inherited from MusicPlayer
phone.browse_internet()   # Defined in Smartphone

File: Python/OOPs/Inheritance/MultipleInheritance.java
package Inheritance;

interface Camera {
    void takePhoto();
}

interface MusicPlayer {
    void playMusic();
}

class Smartphone implements Camera, MusicPlayer {
    public void takePhoto() {
        System.out.println("Photo captured 📷");
    }

    public void playMusic() {
        System.out.println("Playing music 🎵");
    }

    public void browseInternet() {
        System.out.println("Browsing the internet 🌐");
    }
}

public class MultipleInheritance {
    public static void main(String[] args) {
        Smartphone phone = new Smartphone();
        phone.takePhoto();
        phone.playMusic();
        phone.browseInternet();
    }
}
